pkg/gateway/configuration_workingset.go

```diff
@@ -18,16 +18,18 @@ import (
 )
 
 type WorkingSetConfiguration struct {
-	WorkingSet string
-	ociService oci.Service
-	docker     docker.Client
+	WorkingSet   string
```

Comment:
See below, you may just want the WorkingSetConfiguration to have the configuration.  We're certainly not done tweaking this.